rest_server/src/models/marketplace_item.js

```diff
@@ -246,6 +246,14 @@ class MarketplaceItem {
 
   async create(itemReq) {
     const handler = modelSyncHandler(async itemReq => {
+      if (itemReq.isPrivate) {
```

Comment:
>isPrivate [](http://example.com/codeflow?start=18&length=9)

❔ [Question]

Just curious if there is need to have both isPrivate and isPublic. I guess they cannot be true at the same time, so maybe both of them are false?

Reply:
Currently we have 3 different cases here:
1. share to all users (isPublic: true)
2. private (isPrivate: true)
3. share to some specific groups (isPublic: false & isPrivate: false)

In the 3rd case, both of them are true.

Example:
![image](https://user-images.githubusercontent.com/17357108/123902364-57bd8e00-d99f-11eb-9ca6-1d9abc9c63c7.png)

Reply:
> Currently we have 3 different cases here:
> 
> 1. share to all users (isPublic: true)
> 2. private (isPrivate: true)
> 3. share to some specific groups (isPublic: false & isPrivate: false)
> 
> In the 3rd case, both of them are true.

We'd better add this to comments.